sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_client.py

```diff
@@ -91,8 +84,8 @@ def __init__(self, base_url: str, credential: TokenCredential, **kwargs: Any) ->
                 f"Unsupported credential: {type(credential)}. Use an instance of token credential from azure.identity"
             )
         # mypy doesn't compare the credential type hint with the API surface in patch.py
-        self._impl = AzureAppConfiguration(
-            credential, base_url, per_call_policies=self._sync_token_policy, **kwargs  # type: ignore[arg-type]
+        self._impl = AzureAppConfigurationClientGenerated(
+            base_url, credential, per_call_policies=self._sync_token_policy, **kwargs  # type: ignore[arg-type]
```

Comment:
Is the order guaranteed? If not, use keyword arguments.

Reply:
The position params order is reversed in tsp codegen result, the order should be guaranteed if no related updates in tsp or generator.